src/verinfast/config.py

```diff
@@ -423,6 +423,9 @@ def handle_config_file(self):
             if "should_upload" in self.config:
                 self.shouldUpload = self.config["should_upload"]
 
+            if "run_git" in self.config:
+                self.runGit = self.config["run_git"]
+
```

Comment:
I'm not 100% certain how this plays with the ARGS and other places runGit gets set.

Reply:
config is read first and then overwritten by args. runGit is also set true if there's a repo in the conf that isn't local.